### ISSUE TITLE
Xbox One bluetooth controller not correctly mapped to Genesis and PC-Engine

### ISSUE THREAD

**Issue by DanEEStar** [Provenance-Emu/Provenance#1524]
Gamepad controller for some systems seems to be wrongly mapped.

I used the newest self-compiled version of Provenance (develop-branch).
I used a Xbox One controller linked via Bluetooth.

The controls for Genesis and PC-Engine seem to be wrong.
The directional controls are not mapped to the D-Pad but only to the left analog stick. The D-Pad does nothing for Genesis and PC-Engine. Furthermore on Genesis at least one button was not mapped at all to a Gamepad button.
For PC-Engine the buttons seems also not to be mapped to the Gamepad.

I had a previous self-compiled version of Provenance (May/April 2020). There the button mapping for Genesis and PC-Engine was correct.
The directional controls were mapped to the D-Pad and all system buttons were mapped correctly to a Gamepad button.

The controls for PS1 seem to be correct! The controls for the newest version were the same as before.
PS1, Genesis and PC-Engine are the only systems I tested for now.

I would be happy to get some pointers to files which I should investigate. Maybe I can fix the problem myself.

#### APP VERSION
- [x] 1.5 *beta*
- [ ] 1.4

#### APP SOURCE
- [x] Provenance-Emu GitHub  *&nbsp;(Official)*
- [ ] Other  *&nbsp;(Unofficial)*

#### INSTALLED BY 
- [ ] [Side-Loading](https://git.io/vpYzf)
- [x] [Building](https://git.io/vpYzv) *from Source*

#### PLATFORM
- [ ] iPhone
- [x] iPad
- [ ] Apple TV

#### iOS/tvOS VERSION
- [x] 14.x
- [ ] 13.x
- [ ] 12.x
- [ ] 11.x
- [ ] 10.x

-------------------------------------------
<sup> 🚫 We **DO NOT** support unofficial builds installed from 3rd-party sites. ([Official Install](https://git.io/vxABg)) </sup><br><sup> ❓ **Need help or have a suggestion?** Join our [Official Discord](https://discord.gg/4TK7PU5) </sup><br>

**Comment by mrjschulte** [comment]
Try working around it with the Os Remapping option?
Settings > General > Game Controller > Customizations. Then toggle Enable Customizations

**Comment by DanEEStar** [comment]
I discovered the Setting "Controller -> Enable 8BitDo M30 Mapping" while peeking through the code.
This settings was on "true" in my App, I am not sure anymore if I enabled it manually...

When I disable it, then my Xbox One Controller behaved as expected and the button mapping made sense. As in the version from last year.

So I think the ticket can be closed if "false" is the default value for the setting or the settings value should be made "false" by default.
I think more people are using Xbox controllers than 8BitDo M30s...
